docs/vm-images.md

```diff
@@ -13,20 +34,46 @@ is specific requirement for a different OS distribution.
 
 Exceptions:
 
-* [DDN-EXAScaler]: The underlying terraform module has a limitted set of
+* [DDN-EXAScaler]: The underlying terraform module has a limited set of
```

Comment:
what is this limitation, since it is literally the default image for exascaler?

Reply:
Removed this exception since it's not relevant.  See [previous comment on this file](https://github.com/GoogleCloudPlatform/hpc-toolkit/pull/1495/files#r1240068522).